frontend/public/components/RBAC/bindings.jsx

```diff
@@ -455,7 +455,7 @@ const BaseEditRoleBinding = connect(null, {setActiveNamespace: UIActions.setActi
               </Button>
               <Button
                 component="a"
-                to={UIActions.formatNamespacedRouteForResource('rolebindings')}
+                href={UIActions.formatNamespacedRouteForResource('rolebindings')}
```

Comment:
Does this cause the page to reload? We should really be using `Link` instead of `a` elements